clients-simple/src/main/java/uk/gov/gchq/palisade/clients/simpleclient/web/UrlDataClient.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.clients.simpleclient.web;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.openfeign.FeignClientBuilder;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import uk.gov.gchq.palisade.clients.simpleclient.config.ApplicationConfiguration.ClientConfiguration;
+
+import java.util.Map;
+
+/**
+ * The type Url data client for when the profile is not Eureka.
+ */
+@Component
+@Profile("!eureka")
```

Comment:
Double checking on this.  This implies that there are only two possibilities, not Eureka and Eureka.  Could we have more than these two in the future?

Reply:
The profiles we currently have are default, eureka, static and k8s, and the intended bahaviour is to resolve service name via eureka or look in the yamls if you're any of the other profiles (to resolve a url or service-name with k8s discovery). so this covers all edge cases